tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2906,6 +2906,76 @@ Status ConvertConv3D(OpConverterParams* params) {
 Status ConvertConv3DBackpropInputV2(OpConverterParams* params) {
   return ConvertConv3DHelper(params, 1, /*is_conv3d_backprop_input=*/true);
 }
+
+Status ConvertPool3D(OpConverterParams* params) {
```

Comment:
Thanks for the improvement! Could you help to add a test for this?

Reply:
Added tests as requested.

Reply:
Added tests as requested.